taskcluster/ci/build/windows.yml

```diff
@@ -56,10 +56,10 @@ windows/opt:
                 - secrets:get:project/mozillavpn/tokens
             default: []
 
-windows/clang-cl:
+windows/opt:
```

Comment:
This could potentially impact downstream release tasks, but I see they both have the `unsigned.zip` in the `release-artifacts` key.. so I think it should just work